### PR TITLE
Feature add motaroki details

### PR DESCRIPTION
**Pull request by iddle254**
Implemented changes recommended by ruth

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -655,27 +655,44 @@ <h4>Amanda Jepson</h4>
...
             <!--charlesmotaroki@students.uonbi.ac.ke	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/team-5.jpg" class="img-fluid" alt="">
+                  <div class="social">
+                    <a href=""><i class="bi bi-twitter"></i></a>
+                    <a href=""><i class="bi bi-facebook"></i></a>
+                    <a href=""><i class="bi bi-instagram"></i></a>
+                    <a href=""><i class="bi bi-linkedin"></i></a>
```
Comment (RuthNjeri) [inline index.html]: You can add the LinkedIn links and remove the rest of the empty href tags

File: index.html [hunk 2]
```diff
@@ -655,27 +655,44 @@ <h4>Amanda Jepson</h4>
...
         </div>
-          <div class="row"> 
+          <div class="row">
             <!--new change-->
             <!-- awesome code about me-->
             <!--charlesmotaroki@students.uonbi.ac.ke	-->
+            <div class="col-lg-3 col-md-6 d-flex align-items-stretch">
+              <div class="member" data-aos="fade-up" data-aos-delay="400">
+                <div class="member-img">
+                  <img src="assets/img/team/team-5.jpg" class="img-fluid" alt="">
```
Comment (RuthNjeri) [inline index.html:665]: Image should be 600px by 600px for the width and height.

**Comment by RuthNjeri** [comment]
@iddle254  did you see the feedback above?

File: index.html (hunk 1 above)
Comment (iddle254) [reply index.html]: Got it.

File: index.html (hunk 2 above)
Comment (iddle254) [reply index.html:665]: The image is already 600px x 600px

**Comment by RuthNjeri** [comment]
Aside from the extra changes not needed from line 666 in the index.html file, everything else looks good to me.

**Comment by iddle254** [comment]
> Aside from the extra changes not needed from line 666 in the index.html file, everything else looks good to me.

I think these changes are because of prettier

**Pull request merged**